test/integration/resultsPage.js

```diff
@@ -190,4 +190,155 @@ describe('Results page', () => {
       });
     });
   });
+
+  describe('Surgeries with the specific surgery query', () => {
```

Comment:
I think it is worth considering moving these 3 groups of tests out into a new file. This one is almost 350 lines now...